blog/_posts/2016-11-19-call-for-Testing-day-firefox-51b3.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title:  "Firefox 51.0b3 Beta Test day invitation"
+date:   2016-11-19 21:50:00 +0530
+description: Mozilla Tamilnadu Community Testing day invitation
+categories: blog
+tags: [firefoxqa]
+author: prasanthp96
+---
+
+<img height="20%" width="100%" src="https://pbs.twimg.com/media/ChI8q4UU8AAcw9t.jpg">
```

Comment:
Dont add image from outside. Check previous post and see there is image source.. and see the way image added